src/node.cc

```diff
@@ -3398,6 +3402,8 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--v8-options") == 0) {
       new_v8_argv[new_v8_argc] = "--help";
       new_v8_argc += 1;
+    } else if (strncmp(arg, "--v8-pool-size=", 15) == 0) {
+      v8_thread_pool_size = arg + 15;
```

Comment:
I haven’t written much c++ before, but the rationale behind this is that if we parsed the string immediately we would have to set the value of the `int` as it is not a pointer it would have to be `-1` which I am not sure makes sense?

If it does then I can change it to be `-1` if not found and then parse the string immediately.